### DISCUSSION TITLE
multiple monitors Fullscreen borderless

### DISCUSSION THREAD

**Discussion by ageekhere** [OpenRCT2/OpenRCT2#19048] [Feature requests]
A feature that would be good to add is multiple monitor Fullscreen borderless mode, where the game window is stretched across all active monitors while in Fullscreen borderless mode. Normally a work around for this is to use windowed mode and manually stretch the window however it is not full screen.
Here are some code examples that I found to code this.

https://stackoverflow.com/questions/30188182/several-monitors-fullscreen-application-in-qt-4-8
https://www.codeproject.com/Tips/853894/Maximize-Windows-on-Multiple-Monitors

AutoHotkey can be used to change the size of the screen in borderless mode but it would be a hack.

**Comment by IntelOrca** [comment]
For correct multi monitor support, the UI would also need improving so that windows do not open in-between monitors, and the toolbars are more flexible in their positioning.
